### ISSUE TITLE
OFFSPEC Migration - Determine Sample Environment equipment needed

### ISSUE THREAD

**Issue by KathrynBaker** [ISISComputingGroup/IBEX#7297]
As someone using IBEX on OFFSPEC I want to have my Sample Environment Equipment converted for use in IBEX.

## Acceptance Criteria
- [ ] The lists of SE kit for OFFSPEC are updated
- [ ] Tickets are created to undertake any outstanding items

## Notes
- Don't forget to confirm the list with the instrument scientists

**Comment by Tom-Willemsen** [comment]
Note: SM300-5 on offspec is *not* our SM300 motion controller, [it's a power supply](https://telonic.co.uk/product/delta-elektronika-sm300-5-programmable-dc-power-supply-0-300v-0-5a/).

**Comment by Tom-Willemsen** [comment]
Some replies from scientists:
- The various thurlby power supplies are related to spin-echo setup
- SM300-5 (power supply) was used as a backup when something else broke down - scientists unsure what status is currently
- Anything under `delft/` is needed
- "Hydrocell UI" is still a question mark
- Glassman LV is thought to be redundant (and scientist says SDTEST could be used if absolutely necessary to ressurect)
- "ISIS sam env" - scientist says would be nice to retain the functionality but should be low priority
- Offspec BH - scientists think that our existing Keithley + Kepco drivers + a small python script are sufficient to replicate the functionality here. I have asked whether they'd like us to create this script for them and how often they use this
- Thurlby TGA1240 - scientists say this is probably "low priority" but they would like to eventually retain the scientific capability. Jamie has apparently said that best route forwards for this is to support for Tektronix signal generators instead (unclear exactly which model)
- OffSpec Delft RF will need checking to see if it is part of the current SE program.

I will chase a few remaining ambiguous items from this list and then create tickets.

**Comment by Tom-Willemsen** [comment]
Note: for purposes of this ticket, the spin-echo setup has been explicitly declared out-of-scope.

Information about SE drivers on OFFSPEC is at https://github.com/ISISComputingGroup/IBEX/wiki/OFFSPEC-Instrument-Details . There are still a couple of items (mostly hydrocell + DELFT items) that I need to understand fully - some of these may be related to spin-echo but I'm not 100% sure on this yet - have chased scientists and hoping that they will make some time to explain these to me early next week.

Tickets have been written for some items that have been cleared up enough already - these tickets are linked from https://github.com/ISISComputingGroup/IBEX/wiki/OFFSPEC-Instrument-Details

**Comment by Tom-Willemsen** [comment]
Meeting with scientists on beamline scheduled for maintenance day (23rd) to try and understand the remaining kit I am unsure about.

**Comment by Tom-Willemsen** [comment]
Table of SE kit and what I think the IBEX equivalents are is at https://github.com/ISISComputingGroup/IBEX/wiki/OFFSPEC-Instrument-Details

I have proposed one issue - the B-H mapping for magnets - as it is needed for day 1 migration. Everything else that we don't yet support in IBEX is related to the spin-echo / PNR setup, which is out of scope for day 1 migration.

I have created an umbrella issue for spin-echo setup at https://github.com/ISISComputingGroup/IBEX/issues/7491 to help plan the necessary work when OFFSPEC *do* require spin-echo.
